reporter/src/main/kotlin/reporters/AsciiDocTemplateReporter.kt

```diff
@@ -100,6 +101,13 @@ class AsciiDocTemplateReporter : Reporter {
 
                 asciidoctorAttributes.attribute("pdf-theme", themePath)
             }
+            templateOptions.remove(OPTION_PDF_FONTS_DIR)?.let { fontsPath ->
+                File(fontsPath).also {
+                    require(it.isDirectory) { "Could not find pdf-fontsdir path at '${it.absolutePath}'." }
+                }
+
+                asciidoctorAttributes.attribute(OPTION_PDF_FONTS_DIR, "${File(fontsPath).absolutePath},GEM_FONTS_DIR")
+            }
```

Comment:
I have already started a discussion with @neubs-bsi about Asciidoctor attributes. In the future, we might want to add additional attributes and adding an reporter option each time could make the code confusing.
Another approach would be to have an reporter option `asciidoctor-attributes` to which you can give any attribute you want to. However, the `options` variable only maps String to String, so I am not sure how to do this in a clean way.
Furthermore, it would introduce another level of abstraction for the options: ort -> reporter -> asciidoc-reporter -> asciidoc-attributes. 
What are your opinions @neubs-bsi, @sschuberth ?

Reply:
I agree that this could be useful. So far, I don't think it is necessary and would rather do this once we have an actual use case.